### DISCUSSION TITLE
Which is the right way to configure the the secret token?

### DISCUSSION THREAD

**Discussion by m1guer** [lowlighter/metrics#1304] [Q&A] [answered]
i'm trying configure like this `METRICS_TOKEN = <Your GitHub Access Token>` is the right way? if not what is the right way 2 do this?

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !

Yes indeed,

You need to create a personal access token from your setting (use the classic format, the new one hasn't been tested yet)
![image](https://user-images.githubusercontent.com/22963968/205543693-21baf5cf-7e0f-4968-a4ca-95c9042de739.png)

Then put the generated string in your repository secret (don't write it directly in your workflow)
![image](https://user-images.githubusercontent.com/22963968/205543892-71ace7fc-63b2-4a2f-a237-0e8a5a1d3c57.png)

And then it'll be usable with the following syntax in your workflows:
```yml
 with:
   token: ${{ secrets.METRICS_TOKEN }}
```

More informations [here](https://github.com/lowlighter/metrics/blob/master/.github/readme/partials/documentation/setup/action.md#2%EF%B8%8F-put-your-github-personal-token-in-repository-secrets)

**Comment by m1guer** [comment]
i fix that error, my GH_TOKEN syntax was incorrent inside my action secrets
